certbot/certbot/_internal/cli/paths_parser.py

```diff
@@ -46,5 +46,5 @@ def _paths_parser(helpful: "helpful.HelpfulArgumentParser") -> None:
         help=config_help("work_dir"))
     add("paths", "--logs-dir", default=flag_default("logs_dir"),
         help="Logs directory.")
-    add("paths", "--server", default=flag_default("server"),
+    add(["paths", "show_account"], "--server", default=flag_default("server"),
```

Comment:
Mmm, this caught my eye earlier as well, but I wasn't sure whether we should do anything here.

Does making this change mean we should also make the same change for `run`, `certonly`, `register`, `unregister`, `update_account` etc. as well? 

I'm not sure. I think it's worth asking about the thinking behind the `paths` help topic, once people are back from break.

Reply:
Well, I wasn't sure what other options to put there and I didn't know if I could leave it empty :rofl:

Also, if you run e.g. `certbot --help run`, there is no mention of the `--server` option.. Which is kinda weird if you ask me? So I personally think it would be a great idea to add it to those other subcommands too indeed!

Reply:
This can be addressed another time. I'm happy to include it for this verb.